src/esp/geo/OBB.h

```diff
@@ -34,53 +38,60 @@ class OBB {
   }
 
   //! Returns half-extents of this OBB (dimensions)
-  vec3f halfExtents() const { return halfExtents_; }
+  Mn::Vector3 halfExtents() const { return halfExtents_; }
 
   //! Returns quaternion representing rotation of this OBB
-  quatf rotation() const { return rotation_; }
+  Mn::Quaternion rotation() const { return rotation_; }
 
-  //! Return Transform from world coordinates to local [0,1]^3 coordinates
-  const Transform& worldToLocal() const { return worldToLocal_; }
+  //! Return Mn::Matrix4 from world coordinates to local [0,1]^3 coordinates
+  const Mn::Matrix4& worldToLocal() const { return worldToLocal_; }
 
-  //! Return Transform from local [0,1]^3 coordinates to world coordinates
-  const Transform& localToWorld() const { return localToWorld_; }
+  //! Return Mn::Matrix4 from local [0,1]^3 coordinates to world coordinates
+  const Mn::Matrix4& localToWorld() const { return localToWorld_; }
 
   //! Returns an axis aligned bounding box bounding this OBB
-  box3f toAABB() const;
+  Mn::Range3D toAABB() const;
 
   //! Returns distance to p from closest point on OBB surface
   //! (0 if point p is inside box)
-  float distance(const vec3f& p) const;
+  float distance(const Mn::Vector3& p) const;
 
   //! Return closest point to p within OBB.  If p is inside return p.
-  vec3f closestPoint(const vec3f& p) const;
+  Mn::Vector3 closestPoint(const Mn::Vector3& p) const;
 
   //! Returns whether world coordinate point p is contained in this OBB within
   //! threshold distance epsilon
-  bool contains(const vec3f& p, float epsilon = 1e-6f) const;
+  bool contains(const Mn::Vector3& p, float epsilon = 1e-6f) const;
 
   //! Rotate this OBB by the given rotation and return reference to self
-  OBB& rotate(const quatf& rotation);
+  OBB& rotate(const Mn::Quaternion& rotation);
 
  protected:
   void recomputeTransforms();
 
-  vec3f center_;
-  vec3f halfExtents_;
-  quatf rotation_;
-  Transform localToWorld_, worldToLocal_;
+  Mn::Vector3 center_{};
+  Mn::Vector3 halfExtents_{};
```

Comment:
Magnum types are zero-initialized by default (you have to explicitly use `NoInit` to prevent that), no need for the `{}`.